src/software/SfMWebGLViewer/config.h.in

```diff
@@ -0,0 +1,6 @@
+#ifndef _OPENMVG_MAIN_OPENMVG_TO_WEBGL_CONFIG_H_
+#define _OPENMVG_MAIN_OPENMVG_TO_WEBGL_CONFIG_H_
+
+#define OPENMVG_SFM_WEBGL_PATH "@OPENMVG_SOFTWARE_SFMWEBGLVIEWER_SRC_DIR@"
```

Comment:
There is no way to provide a path for this using install mode ? Or maybe provide two paths : one for the build and one for the install, and the main manage the correct path (the first one that contains everything ?)

I think, it's not user friendly and maybe too openMVG-internal centric to be provided as an option.